testcrawl.csv

```diff
@@ -0,0 +1,11 @@
+False,Commission,,2024-07-08 20:00:00,losca_health_commission/202407081800/x/los_angeles_city_health_commission_meeting,"[{'href': None, 'title': 'videoLink'}, {'href': 'https://lacity.primegov.com/Portal/Meeting?meetingTemplateId=131218', 'title': 'HTML Notice of Cancellation'}, {'href': 'https://lacity.primegov.com/Public/CompiledDocument?meetingTemplateId=131218&compileOutputType=1', 'title': 'Notice of Cancellation'}]","{'address': '200 N Spring St, Room 340 (CITY HALL) Los Angeles, CA 90012', 'name': 'Los Angeles City Health Commission'}",https://lacity.primegov.com/api/v2/PublicPortal/ListArchivedMeetingsByCommitteeId?year=2024&committeeId=6,2024-07-08 18:00:00,passed,,Los Angeles City Health Commission Meeting
+False,Commission,,2024-08-12 20:00:00,losca_health_commission/202408121800/x/los_angeles_city_health_commission,"[{'href': 'https://youtube.com/watch?v=n7Yv4_tL5Hk', 'title': 'videoLink'}, {'href': 'https://lacity.primegov.com/Portal/Meeting?meetingTemplateId=131577', 'title': 'HTML Revised Agenda'}, {'href': 'https://lacity.primegov.com/Public/CompiledDocument?meetingTemplateId=131577&compileOutputType=1', 'title': 'Revised Agenda'}]","{'address': '200 N Spring St, Room 340 (CITY HALL) Los Angeles, CA 90012', 'name': 'Los Angeles City Health Commission'}",https://lacity.primegov.com/api/v2/PublicPortal/ListArchivedMeetingsByCommitteeId?year=2024&committeeId=6,2024-08-12 18:00:00,passed,,Los Angeles City Health Commission
+False,Commission,,2024-09-09 20:00:00,losca_health_commission/202409091800/x/los_angeles_city_health_commission,"[{'href': 'https://youtube.com/watch?v=E_sQC_pw-N4', 'title': 'videoLink'}, {'href': 'https://lacity.primegov.com/Portal/Meeting?meetingTemplateId=132502', 'title': 'HTML Agenda'}, {'href': 'https://lacity.primegov.com/Public/CompiledDocument?meetingTemplateId=132502&compileOutputType=1', 'title': 'Agenda'}]","{'address': '200 N Spring St, Room 340 (CITY HALL) Los Angeles, CA 90012', 'name': 'Los Angeles City Health Commission'}",https://lacity.primegov.com/api/v2/PublicPortal/ListArchivedMeetingsByCommitteeId?year=2024&committeeId=6,2024-09-09 18:00:00,passed,,Los Angeles City Health Commission
+False,Commission,,2024-10-21 20:00:00,losca_health_commission/202410211800/x/los_angeles_city_health_commission,"[{'href': 'https://youtube.com/watch?v=BfGPNNhZM5Q', 'title': 'videoLink'}, {'href': 'https://lacity.primegov.com/Portal/Meeting?meetingTemplateId=134366', 'title': 'HTML Special Agenda'}, {'href': 'https://lacity.primegov.com/Public/CompiledDocument?meetingTemplateId=134366&compileOutputType=1', 'title': 'Special Agenda'}]","{'address': '200 N Spring St, Room 340 (CITY HALL) Los Angeles, CA 90012', 'name': 'Los Angeles City Health Commission'}",https://lacity.primegov.com/api/v2/PublicPortal/ListArchivedMeetingsByCommitteeId?year=2024&committeeId=6,2024-10-21 18:00:00,passed,,Los Angeles City Health Commission
```

Comment:
Even though this test file is a lot smaller than the other one, we should still remove this file.